mdn/kuma
# Create Subscriptions landing page

**Summary:** 
Convert existing FAQ page to the Subscription landing page in React. 

**To do:** 
- [ ] Translations are not working (see screenshots below, where locale is `FR` and header is translated, but not the main content.

**To test:** 
Go to /payments (http://localhost.org:8000/en-US/payments): 
<img width="1223" alt="Screen Shot 2020-03-05 at 5 29 40 PM" src="https://user-images.githubusercontent.com/650/76032089-2053e400-5f07-11ea-8297-9d49b03ef157.png">

Make sure former /payments/recurring page still works by going to /payments/recurring:
<img width="1228" alt="Screen Shot 2020-03-05 at 5 29 47 PM" src="https://user-images.githubusercontent.com/650/76032084-1cc05d00-5f07-11ea-80be-f065d5c71683.png">
 
Closes #6582
